sdk/keyvault/keyvault-admin/README.md

```diff
@@ -34,16 +33,160 @@ TypeScript users need to have Node type definitions installed:
+
+#### Create KeyVaultAccessControlClient
+
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultAccessControlClient][src-rbac-client]:
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyVaultBackupClient } from "@azure/keyvault-admin";
+
+const credentials = new DefaultAzureCredential();
+
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const client = new KeyVaultAccessControlClient(vaultUrl, credentials);
+```
+
+#### Create KeyVaultBackupClient
 
-### Configuring your Key Vault
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** 
+with the above returned URI, you can create the [KeyVaultBackupClient][src-backup-client]:
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { KeyVaultBackupClient } from "@azure/keyvault-admin";
+
+const credentials = new DefaultAzureCredential();
+
+const vaultUrl = `https://<MY KEY VAULT HERE>.vault.azure.net`;
+const client = new KeyVaultBackupClient(vaultUrl, credentials);
+```
 
 ## Key concepts
 
-## Authenticating with Azure Active Directory
+### KeyVaultRoleDefinition
+
+A Role Definition is a collection of permissions. A role definition defines the operations that can be performed, such as read, write, 
+and delete. It can also define the operations that are excluded from allowed operations.
+
+Role definitions can be listed and specified as part of a Role Assignment.
+
+### KeyVaultRoleAssignment. 
+
+A Role Assignment is the association of a Role Definition to a service principal. They can be created, listed, fetched individually, and deleted.
+
+### KeyVaultAccessControlClient
+
+A [KeyVaultAccessControlClient][src-rbac-client] provides both synchronous and asynchronous operations allowing for management of Role Definitions (instances of `KeyVaultRoleDefinition`) and Role Assignments (instances of `KeyVaultRoleAssignment`).
+
+### KeyVaultBackupClient
+
+A [KeyVaultBackupClient][src-backup-client] provides both synchronous and asynchronous operations for performing full key backups, full key restores, and selective key restores.
+
+### Long Running Operations
+
+The Azure SDK for TypeScript and JavaScript offers a common abstraction for long running operations through the package [core-lro][core-lro]. In particular, the operations done by the [KeyVaultBackupClient][src-backup-client] may take as much time as needed by the Azure resources, requiring a client layer to keep track, serialize and resume the operations through the lifecycle of the programs that wait for them to finish.
```

Comment:
```suggestion
The Azure SDK for TypeScript and JavaScript offers a common abstraction for long running operations through the package [@azure/core-lro][core-lro]. In particular, the operations done by the [KeyVaultBackupClient][src-backup-client] may take as much time as needed by the Azure resources, requiring a client layer to keep track, serialize and resume the operations through the lifecycle of the programs that wait for them to finish.
```

Reply:
I think we should mention it. The Poller methods aren't part of the client's public API. The public API file imports from core-lro:

https://github.com/Azure/azure-sdk-for-js/pull/11009/files#diff-d1d574cf276744fd53761f9a30b41d49R9-R10

Reply:
Well, the question is whether the user needs to import the core-lro package in order to use these long running operations. If no, then I would recommend not mentioning it here.

Even if we do want to mention it, I would put the line on it in the end and not attract too much attention to it. We want to draw attention to the fact that there are long running operations in this package and these methods are different from the others. The core lro package is implementation details

```suggestion
The operations done by the [KeyVaultBackupClient][src-backup-client] may take as much time as needed by the Azure resources, requiring a client layer to keep track, serialize and resume the operations through the lifecycle of the programs that wait for them to finish. This is done via a common abstraction through the package [@azure/core-lro][core-lro]. 
```

Reply:
> Well, the question is whether the user needs to import the core-lro package in order to use these long running operations. If no, then I would recommend not mentioning it here.

No, they don't need to import this package.

I like that update! I'll add it.